docs/src/content/docs/book/contracts.mdx

```diff
@@ -538,6 +538,42 @@ Read more about them in their dedicated section: [Internal functions](/book/func
 
 :::
 
+### Converting to Cell or Slice
+
+<Badge text="Available since Tact 1.6.12" variant="tip" size="medium"/><p/>
+
+You can convert a contract instance to a [`Cell{:tact}`](/book/cells#cells) or [`Slice{:tact}`](/book/cells#slices) using the `toCell(){:tact}` and `toSlice(){:tact}` methods.
+These methods will build a new `Cell{:tact}` or `Slice{:tact}` with the contract data.
+
+```tact
+contract Example(
```

Comment:
Let's reuse our implementation of Wallet V4 as an example here. Just a small part of it

Reply:
Done